_posts/2020-01-29-cognitive-maps.md

```diff
@@ -0,0 +1,196 @@
+  </figure>
+</div>
+
+A recent opinion piece in the Washington Post titled, [“Ditch the GPS. It’s ruining your brain.”](https://www.washingtonpost.com/opinions/ditch-the-gps-its-ruining-your-brain/2019/06/05/29a3170e-87af-11e9-98c1-e945ae5db8fb_story.html) sparked some debate on this topic. In this piece, science writer M.R. O’Connor cautions that extensive use of GPS navigation affects our perception and judgment. He continues to say that this dependency disconnects us from our surrounding environment as we follow routes that we don’t remember.
+
+A 2017 article in the journal Nature compared the activity in the hippocampus between a group who navigated with GPS and a group who navigated without it. The study found that there was more activity in the hippocampus in the group who navigated without GPS (Javadi et al 2017).
+
+But, neuroscientist Kate Jeffery, thinks it’s unlikely that extensive use of GPS will result in hippocampal atrophy. She points out that the hippocampus is still pretty busy as people today spend a lot more time now navigating online maps and virtual environments (Advisory Board 2019).
+
+While the verdict is still out on the long term effects of GPS-based navigation, there is plenty of evidence that suggests traditional way-finding is good for our brains. Bohbot said, “When we get lost, it activates the hippocampus, it gets us completely out of the habit mode. Getting lost is good!”.
+
+The brain connections that we make when we explore and navigate our environment help us better connect to the world around while nourishing what is called topophilia - “a strong sense of place, which often becomes mixed with the sense of cultural identity among certain people and a love of certain aspects of such a place” (Tuan 1974).
+
+### Organizing Life's Experiences
+{: .text-left}
+
+We now know that our brain is a spatial organ. The hippocampus, in particular, is responsible for mapping, navigation, and memory storage. These functions are linked to additional mechanisms such as episodic and long term memories. Research has shown that when we exercise these functions we see an increase in brain activity. These active neurons not only create new connections but preserve and organize existing ones (Javadi et al 2017).
+
+At the core of these functions is a system that stores and retrieves spatial information. In this context, it's easy to make a comparison between the brain a GIS. The hippocampus acts much like a file indexing system working with other parts of the brain, that function as a database, making transactions. When we add the episodic memory aspect, it's similar to enabling the spatial component on the database: memories now contain a geographic location. This explains why places, --such as a bend in the road-- or an event --such as September 11th-- can trigger the retrieval of certain memories. When we think of September 11th, 2001, we not only remember the event but also the where-we-were.
```

Comment:
should a be and in the sentence `In this context, it's easy to make a comparison between the brain a GIS.`?

```suggestion
At the core of these functions is a system that stores and retrieves spatial information. In this context, it's easy to make a comparison between the brain and GIS. The hippocampus acts much like a file indexing system working with other parts of the brain, that function as a database, making transactions. When we add the episodic memory aspect, it's similar to enabling the spatial component on the database: memories now contain a geographic location. This explains why places, --such as a bend in the road-- or an event --such as September 11th-- can trigger the retrieval of certain memories. When we think of September 11th, 2001, we not only remember the event but also the where-we-were.
```